Sign Up Page/signup.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./signup.css" />
+    <title>Sign Up Page</title>
+  </head>
+  <body>
+    <div class="maincon">
+      <h1>Welcome</h1>
+      <h2>Create Account</h2>
+      <form action="./SignUp.html">
+        <div class="namecon">
+          <input
+            name="user_name"
+            required
+            id="user_name"
+            placeholder="Name"
+            type="text"
+            autocomplete="off"
+          />
+        </div>
+
+        <div class="mailcon">
+          <input
+            required
+            name="email"
+            id="email"
+            placeholder="Your Email"
+            type="email"
+            autocomplete="off"
+          />
+        </div>
+
+        <div class="passcon">
+          <input
+            name="password"
+            required
+            id="password"
+            placeholder="Password"
+            type="password"
+            autocomplete="off"
```

Comment:
in every input the autocomplete option is by default off so why are you writing it again and again?

Reply:
It was showing the drop down menu of previous inputs which I didn't want as they were just inputs used for testing.